addons/html_builder/static/src/core/media_website_plugin.js

```diff
@@ -65,14 +67,59 @@ export class MediaWebsitePlugin extends Plugin {
                 this.onDblClickEditableMedia(targetEl);
             }
         });
+
+        this.showTooltip = false;
+        this.popover = this.services.popover;
+        this.tooltipTimeouts = [];
+        this.removeCurrentTooltip = undefined;
+
+        this.addDomListener(this.editable, "click", (ev) => {
+            const targetEl = ev.target.closest(mediaSelector);
+            if (!targetEl) {
+                return;
+            }
+            let isEditable = (targetEl.parentElement && targetEl.parentElement.isContentEditable);
+
+            if (!isEditable && targetEl.classList.contains("o_editable_media")) {
+                isEditable = shouldEditableMediaBeEditable(targetEl);
+            }
+            if (
+                isEditable &&
+                !isProtected(this.dependencies.selection.getEditableSelection().anchorNode)
+            ) {
+                this.onMousedownEditableMedia(targetEl);
+            }
+        });
     }
 
     onDblClickEditableMedia(mediaEl) {
+        //do not show tooltip on double click
+        if (this.removeCurrentTooltip) {
+            this.removeCurrentTooltip();
+        }   
         const params = { node: mediaEl };
         const sel = this.dependencies.selection.getEditableSelection();
 
         const editableEl =
             closestElement(params.node || sel.startContainer, ".o_editable") || this.editable;
         this.dependencies.media.openMediaDialog(params, editableEl);
     }
+
+    onMousedownEditableMedia(mediaEl) {
+        //delete last tooltip if it's still displayed
+        if (this.removeCurrentTooltip) {
+            this.removeCurrentTooltip();
+        }   
```

Comment:
2 solutions:
- in the setup, this.removeCurrentTooltip = () => {}, you can remove the if with this solution;
- when you removeCurrentTooltip, you should to make it undefined again.